1-JavaScript/Week3/homework/ex3-tellFortune.js

```diff
@@ -33,29 +33,53 @@ body, this code is now written once only in a separated function.
 
 // This function should take an array as its parameter and return
 // a randomly selected element as its return value.
-function selectRandomly(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function selectRandomly(myArray) {
+  
+  const randomElementOfArray =
+    myArray[Math.floor(Math.random() * myArray.length)];
+  return randomElementOfArray;
```

Comment:
```suggestion
  return myArray[Math.floor(Math.random() * myArray.length)];
```
I would opt for returning the return value of this expression, because we're not using randomElementOfArray anywhere else and its expressive enough to be easily understood without the need for some naming to explain it.

Reply:
I changed the return value